mne/channels/tests/test_interpolation.py

```diff
@@ -439,3 +439,64 @@ def test_method_str():
         raw.interpolate_bads(method="spline")
     raw.pick("eeg", exclude=())
     raw.interpolate_bads(method="spline")
+
+
+@pytest.mark.parametrize("montage_name", ["biosemi16", "standard_1020"])
```

Comment:
Some test that checks the order would be good, too. Like have one MEG/EOG/ECG or whatever channel as the first channel, followed by the EEG channels, followed by some other MEG/EOG/ECG channel; then interp; then check that the first channel (non-EEG) is still first (and has data that is unchanged), and the last channel is still last (and has data that is unchanged).